huaweicloud/services/acceptance/vpc/resource_huaweicloud_networking_secgroup_rule_test.go

```diff
@@ -251,8 +252,8 @@ func TestAccNetworkingSecGroupRule_priority(t *testing.T) {
 }
 
 func testAccCheckNetworkingSecGroupRuleDestroy(s *terraform.State) error {
```

Comment:
```suggestion
	rc := acceptance.InitResourceCheck(
		resourceName,
		&ar,
		xxx,
	)
```

replace checkDestroy and check exit with this function

Reply:
fixed